jetty-core/jetty-io/src/test/java/org/eclipse/jetty/io/ByteBufferAccumulatorTest.java

```diff
@@ -282,47 +292,35 @@ private int writeInFlushMode(ByteBuffer from, ByteBuffer to)
         return written;
     }
 
-    public static class CountingBufferPool extends LeakTrackingByteBufferPool
+    private static class CountingBufferPool extends RetainableByteBufferPool.Wrapper
```

Comment:
This pattern has been replicated across quite a few tests. Shouldn't we introduce it as a non-test class and use it everywhere there is a `TODO: track leaks`?

It's not a full blown leak tracker, but it could at least be used to verify that tests aren't leaking.

Reply:
Sure, but in another PR.